context/app/static/js/helpers/trackers.js

```diff
@@ -0,0 +1,65 @@
+import MatomoTracker from '@datapunt/matomo-tracker-js';
+import ReactGA from 'react-ga';
+
+function getSiteId(location) {
+  const { host } = location;
+
+  // siteIds should correspond to the list at:
+  // https://hubmapconsortium.matomo.cloud/index.php?module=SitesManager
+
+  switch (host) {
+    case 'portal.hubmapconsortium.org':
+      return 1;
+    case 'localhost:5001':
+      return 2;
+    default:
+      return 3;
+  }
+}
+
+const tracker = new MatomoTracker({
+  urlBase: 'https://hubmapconsortium.matomo.cloud/',
+  // eslint-disable-next-line no-restricted-globals
+  siteId: getSiteId(location),
+  // userId: 'UID76903202', // optional, default value: `undefined`.
+  // trackerUrl: 'https://LINK.TO.DOMAIN/tracking.php', // optional, default value: `${urlBase}matomo.php`
+  // srcUrl: 'https://LINK.TO.DOMAIN/tracking.js', // optional, default value: `${urlBase}matomo.js`
+  // disabled: false, // optional, false by default. Makes all tracking calls no-ops if set to true.
+  // heartBeat: { // optional, enabled by default
+  //   active: true, // optional, default value: true
+  //   seconds: 10 // optional, default value: `15
+  // },
+  // linkTracking: false, // optional, default value: true
+  // configurations: { // optional, default value: {}
+  //   // any valid matomo configuration, all below are optional
+  //   disableCookies: true,
+  //   setSecureCookie: true,
+  //   setRequestMethod: 'POST'
+  // }
+});
+
+ReactGA.initialize('UA-133341631-3', { testMode: process.env.NODE_ENV === 'test' });
```

Comment:
I think this still needs to be initialized at the `App` level?

Reply:
@john-conroy :
> I think this still needs to be initialized at the App level?

I could be confused, but I don't think so? I can double check, but I think events were getting logged... but even if it does work, that doesn't mean it's the best way to do things...

The two are a little different: We initialize `ReactGA` directly, while we need to create a new instance of `MatomoTracker`. But in either case, their methods are only called inside this module, so I don't think there's any need to expose these internals anywhere else... In fact, the opposite: one point of this PR is to hide these details from the rest of the codebase.

Talk after standup if this doesn't seem right?

Reply:
We're not absolutely sure about this, but got John on-board to try it out: He points out that usual guidance is to initialize early on... GA may do some logging beyond what we explicitly call for? But the module private consts will be initiated when the module is loaded, which seems early enough.